packages/memory/src/utils/uint8array-to-string.ts

```diff
@@ -0,0 +1,6 @@
+const decoder = new TextDecoder();
+
+/**
+ * Convert a JS Uint8Array to a String
+ */
+export const uint8ArrayToString = (val: Uint8Array) => decoder.decode(val);
```

Comment:
Do we perhaps need a check for whether the IntArray actually is UTF-8 encoded? or is this not a risk we have to care about? 🤔 

Situations like:
```js
decoder.decode(new Uint8Array([0, 0, 0, 0, 0])) => "\u0000\u0000\u0000\u0000\u0000" 
```
```js
decoder.decode(new Uint8Array([207, 240, 232, 226, 229, 242, 44, 32, 236, 232, 240, 33])) => "������, ���!"
```

Reply:
The second example is still a string though! Sure it's a garbled mess, but you could re-encode that back to it's original Uint8Array

Reply:
Both are still string definitely, that why i phrased it as question 😄 
The second one makes me worry about keys collisions as we're using these string for indexing an object.
```js
dec.decode(new Uint8Array([232, 240])) === dec.decode(new Uint8Array([207, 226])) 
```

Reply:
Since the values are set by ourselves, I don't think we have to take this into account 👍